README.md

```diff
@@ -40,13 +40,19 @@ private_movie_views = \
 
 # Calculate the private sum of ratings per movie
 dp_result = private_movie_views.sum(
-    SumParams(max_partitions_contributed=2,
-              max_contributions_per_partition=2,
+    SumParams(
+              # Limits to how much one user can contribute:
+              # .. at most two movies rated per user
+              max_partitions_contributed=2,
+              # .. at most one ratings for each movie
+              max_contributions_per_partition=1,
+              # .. with minimal rating of "1"
               min_value=1,
+              # .. and maximum rating of "5"
               max_value=5,
-              # Specifies the aggregation key
+              # The aggregation key: we're grouping data by movies
```

Comment:
What do you think about moving extractors to the first place (in examples, not in function definition), that it's more logical:
1.We define partition and value
2.Limit per partition
3.Limit values

Reply:
Good idea, done.